backend.native/tests/build.gradle

```diff
@@ -3876,6 +3881,53 @@ if (isAppleTarget(project)) {
         }
         swiftSources = ['framework/multiple/multiple.swift']
     }
+
+    task testGh3343Framework(type: FrameworkTest) {
+        final String frameworkName = 'Gh3343'
+        testName = frameworkName
+        frameworkNames = [frameworkName]
+
+        def mainTask = it
+        task prebuild(type: DefaultTask) {
+            execKonanClang(project.target) {
+                args "$projectDir/framework/gh3343/objclib.m"
+                args "-lobjc", '-fobjc-arc'
+                args '-fPIC', '-shared', '-o', "$buildDir/libobjcgh3343.dylib"
+            }
+            if (project.target instanceof KonanTarget.IOS_X64) {
+                UtilsKt.codesign(project, "$buildDir/libobjcgh3343.dylib")
+            }
+        }
+
+        konanArtifacts {
+            def lib = "objcGh3343"
+            def libTask = "compileKonan${lib.capitalize()}${target.name.capitalize()}"
+            UtilsKt.dependsOnDist(project, libTask)
+            UtilsKt.sameDependenciesAs(libTask, mainTask)
+            mainTask.dependsOn(libTask)
+            project.tasks.getByName(libTask).dependsOn("prebuild")
+
+            def lnOpts = (project.tasks.getByName(libTask)).linkerOpts
+
+            framework(frameworkName, targets: [ target ]) {
+                libraries {
+                    artifact lib
+                }
+                srcDir 'framework/gh3343'
+                baseDir "$testOutputFramework/$testName"
+
+                if (!useCustomDist) {
+                    dependsOn ":${target}CrossDistRuntime", ':commonDistRuntime', ':distCompiler'
+                }
+
+                linkerOpts lnOpts
+                extraOpts "-Xembed-bitcode-marker", "-Xobjc-generics"
```

Comment:
Does adding `-Xobjc-generics` make any difference here? IIRC, this flag even no longer exists.